auth-lib/src/main/AndroidManifest.xml

```diff
@@ -44,7 +44,6 @@
         </activity>
 
         <activity
-            android:launchMode="singleInstance"
```

Comment:
I am not sure why it was set before, but I am pretty sure it was to fix some other bug. 
Let live without it for now and please run an extensive manual testing session trying to login from different activities and other weird cases.